compiler/test/dotty/tools/dotc/EntryPointsTest.scala

```diff
@@ -20,9 +21,9 @@ class EntryPointsTest {
   private val sources =
     List("../tests/pos/HelloWorld.scala").map(p => new java.io.File(p).getPath())
   private val dottyInterfaces =
-    new java.io.File("../interfaces/dotty-interfaces-0.1-SNAPSHOT.jar").getPath
+    new java.io.File(Jars.dottyInterfaces).getPath
   private val dottyLibrary =
-    new java.io.File("../library/target/scala-2.11/dotty-library_2.11-0.1-SNAPSHOT.jar").getPath
+    new java.io.File(Jars.dottyLib).getPath
```

Comment:
Note that the whole point of this test and `InterfaceEntryPointTest` is to give an example of how to use the compilers entry points for external projects; so relying on extra files and/or hardcoded path is not great.